app/models/score_manager.rb

```diff
@@ -9,29 +9,24 @@ def initialize(user, habits, num_days_ago)
   end
 
   def daily_scores
-    day_score = 0
-    if @num_days_ago > 0
-      date = @num_days_ago.days.ago.localtime
-    else
-      date = DateTime.now
-    end
-    weekday = date.strftime('%A')
-    if check_todays_score(date).empty?
+    @day_score = 0
+    @date = calculate_date
+    weekday = @date.strftime('%A')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.